webview/src/experiments/components/table/CellHintTooltip.tsx

```diff
@@ -7,20 +7,24 @@ import Tooltip, {
 
 export type CellHintTooltipProps = {
   tooltipContent: ReactNode
+  tooltipOffset?: [number, number]
   children: ReactElement
 }
 
 export const CellHintTooltip: React.FC<CellHintTooltipProps & TippyProps> = ({
   tooltipContent,
   children,
-  delay = NORMAL_TOOLTIP_DELAY
+  delay = NORMAL_TOOLTIP_DELAY,
+  tooltipOffset = [0, -2]
 }) => {
   return (
     <Tooltip
       content={<span className={styles.cellHintTooltip}>{tooltipContent}</span>}
+      appendTo={document.body}
```

Comment:
I'm struggling a bit in a similar situation (finally got to the tooltip improvements for the Plots webview - to add links, etc). In my case click on tooltip triggers the onClick on the `summary` element and toggles the whole section - super annoying. May be @mattseddon is trying to avoid the same situation here? In my case though `appendTo` didn't help. If someone has experience please let me know how to overcome this. (I'll try to figure this out later today otherwise).

Reply:
Tooltip was hidden behind the rows. I'll post the before and after screenshot and source of the fix.

Reply:
Not sure without seeing the code, but usually this can be solved by using the parameter of the `onClick` function to stop the propagation of the `onClick` to parent elements

```
onClick = (e: MouseEvent<HTMLElement>) => {
  e.stopPropagation()
  ...
}
```

Reply:
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/37993418/196257894-378a3f2a-f44d-4da6-817d-784a93910902.png">

https://stackoverflow.com/questions/62405564/tippy-js-z-index-not-working-with-interactivity

Reply:
Tried stop `e.stopPropagation`, no change when the tooltip is interactive. We can pair on it today.